network-api/networkapi/wagtailpages/pagemodels/customblocks/recent_blog_entries.py

```diff
@@ -50,25 +58,16 @@ def get_context(self, value, parent_context=None):
         topic = value.get("topic_filter", False)
 
         # default filter and query
-        type = "tags"
+        query_type = "tags"
```

Comment:
**Kudos** Thanks for this update @AdalbertoMoz this definitely makes the logic more readable and easy to understand 🙌